deploy/helm/airflow-operator/crds/crds.yaml

```diff
@@ -591,6 +591,10 @@ spec:
                           - repo
                         type: object
                       type: array
+                    dbInit:
+                      default: true
+                      description: Whether to execute the database initialization routines (a combination of database initialization, upgrade and migration depending on the Airflow version). Defaults to true to be backwward compatible.
```

Comment:
During decision talks we said that we likely want something like "Setting this to false is unsafe territory, you need to know what you are doing. We can not support you with problems related to this when it's turned off"